commercetools.Sdk/commercetools.Sdk.Domain/Common/IShopping.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using commercetools.Sdk.Domain.CartDiscounts;
+using commercetools.Sdk.Domain.Carts;
+using commercetools.Sdk.Domain.CustomerGroups;
+using commercetools.Sdk.Domain.Orders;
+using commercetools.Sdk.Domain.Stores;
+
+namespace commercetools.Sdk.Domain.Common
+{
+    public interface IShopping
```

Comment:
The JVM SDK uses CartLike as a common interface name. Would be nice if you could introduce a ICartLike too and deprecate the IShopping one ;)